scripts/actions/list_all_pads

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC STARTING
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source scripts/_common.sh
+source /usr/share/yunohost/helpers
+
+#=================================================
+# MANAGE SCRIPT FAILURE
+#=================================================
+
+# Exit if an error occurs during the execution of the script
+ynh_abort_if_errors
+
+#=================================================
+# RETRIEVE ARGUMENTS
+#=================================================
+
+app=${YNH_APP_INSTANCE_NAME:-$YNH_APP_ID}
+
+db_name=$(ynh_app_setting_get --app=$app --key=db_name)
+db_pwd=$(ynh_app_setting_get --app=$app --key=mysqlpwd)
+
+#=================================================
+# CHECK IF ARGUMENTS ARE CORRECT
+#=================================================
+
+#=================================================
+# CHECK IF AN ACTION HAS TO BE DONE
+#=================================================
+
+#=================================================
+# SPECIFIC ACTION
+#=================================================
+# LIST ALL PADS FROM THE DATABASE
+#=================================================
+
+ynh_script_progression --message="Listing all pads..." --weight=3
+
+mysql -u $db_name -p$db_pwd $db_name --silent -e 'select distinct substring(store.key,5,locate(":",store.key,5)-5) as "pads" from store where store.key like "pad:%"' | sed 's/^/>> /g' >&2
```

Comment:
Why you don't use `ynh_mysql_connect_as`?
Are the `--silent -e` parameters really necessary? (<sub><sup>don't hurt me :x</sup></sub>)

Reply:
Too old for me to remember... But I'm going to investigate that.

Reply:
`-e` (`--execute=statement`) is what we should use instead of that redirection on stdin.
This is not clear, not practical, and absolutely not safe, put that into a while loop and you can be sure it will fail...

Anyway...I reworked that line with the helper.
Running some tests before I commit.